src/Microsoft.SymbolStore/KeyGenerators/PDBFileKeyGenerator.cs

```diff
@@ -33,7 +33,10 @@ public override IEnumerable<SymbolStoreKey> GetKeys(KeyTypeFlags flags)
             {
                 if ((flags & KeyTypeFlags.IdentityKey) != 0)
                 {
-                    yield return GetKey(_path, _pdbFile.Signature, unchecked((int)_pdbFile.DbiAge));
+                    if (_pdbFile.DbiStream.IsValid())
+                    {
+                        yield return GetKey(_path, _pdbFile.Signature, unchecked((int)_pdbFile.DbiAge));
```

Comment:
Will not generating a key for a generator result in an exception somewhere, or will uptaking this in the uploader be safe?

Reply:
No exception. It just doesn't add it to the list of keys to be returned to the symbol uploader.